plugins/modules/system/make.py

```diff
@@ -40,6 +40,11 @@
       - Use a specific make binary.
     type: path
     version_added: '0.2.0'
+  jobs:
+    description:
+      - Set the number of make jobs to run concurrently.
+      - Typically if set, this would be the number of processors and/or threads available to the machine
```

Comment:
```suggestion
      - Typically if set, this would be the number of processors and/or threads available to the machine.
      - This is not supported by all make implementations.
```